account_budget_oca/views/account_budget_views.xml

```diff
@@ -0,0 +1,310 @@
+                                            </group>
+                                            <group>
+                                                <label for="date_from" string="Period"/>
+                                                <div>
+                                                    <field name="date_from" class="oe_inline"/> - 
+                                                    <field name="date_to" class="oe_inline"/>
+                                                </div>
+                                                <field name="paid_date" groups="base.group_no_one"/>
+                                                <field name="company_id" options="{'no_create': True}" groups="base.group_multi_company"/>
+                                            </group>
+                                        </group>
+                                    </form>
+                                </field>
+                            </page>
+                        </notebook>
+                    </sheet>
+                    <div class="oe_chatter">
+                        <field name="message_follower_ids" widget="mail_followers" groups="base.group_user"/>
+                        <field name="message_ids" widget="mail_thread"/>
+                    </div>
+                </form>
+            </field>
+        </record>
+
+        <record model="ir.ui.view" id="crossovered_budget_view_tree">
+            <field name="name">crossovered.budget.view.tree</field>
+            <field name="model">crossovered.budget</field>
+            <field name="arch" type="xml">
+               <tree decoration-info="state == 'draft'" decoration-muted="state in ('done','cancel')" string="Budget">
+                    <field name="name" colspan="1"/>
+                    <field name="date_from"/>
+                    <field name="date_to"/>
+                    <field name="company_id" options="{'no_create': True}" groups="base.group_multi_company"/>
+                    <field name="creating_user_id"/>
+                    <field name="state"/>
+                </tree>
+            </field>
+        </record>
+
+        <record id="view_crossovered_budget_kanban" model="ir.ui.view">
+            <field name="name">crossovered.budget.kanban</field>
+            <field name="model">crossovered.budget</field>
+            <field name="arch" type="xml">
+                <kanban class="o_kanban_mobile">
+                    <field name="name"/>
+                    <field name="date_from"/>
+                    <field name="date_to"/>
+                    <field name="creating_user_id"/>
+                    <field name="state"/>
+                    <templates>
+                        <t t-name="kanban-box">
+                            <div t-attf-class="oe_kanban_global_click">
+                                <div class="row mb4">
```

Comment:
The kanban is not displayed correctly. Could you check?

![Screenshot from 2019-05-29 09-45-55](https://user-images.githubusercontent.com/7657635/58539065-d6f79b00-81f6-11e9-9886-c222a76053ae.png)

Reply:
Looks good. Thanks! :+1: